piksi_tools/console/settings_view.py

```diff
@@ -212,6 +227,40 @@ def _factory_default_button_fired(self):
                         + "Are you sure you want to reset to factory defaults?"
     confirm_prompt.run(block=False)
 
+  def _settings_save_to_file_fired(self):
```

Comment:
Cool. Maybe we can print a very loud, obvious message when the baud rate is changed on any of the USARTs?